af_data_jobseeker_api_loader/__manifest__.py

```diff
@@ -15,7 +15,7 @@
     "website": "https://vertel.se/",
     "category": "Tools",
     "depends": [ 
-		"edi_af_aisf_rask_get_jobseeker", 
+		"af_aisf_jobseeker_sync",
```

Comment:
also update version number and description

Reply:
Changed version from 12.0.1.1 to 12.0.1.2 and added version-value to description attribute